src/lib/mapper.js

```diff
@@ -38,6 +43,33 @@ export default class Mapper {
 
   }
 
+  // execute(source, destination) {
```

Comment:
Well... I have so much anger  in my black heart for commented out code. This is why we use version tracking...

But again... Religion...